### PR TITLE
feat: better multipart tests

### PR DESCRIPTION
**Pull request by loks0n**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: mock-server/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ services:
   mockapi:
     container_name: mockapi
     ports:
-      - 8080:80
+      - 3175:80
```
Comment (Meldiron) [inline mock-server/docker-compose.yml:5]: Is this change needed?

File: tests/Base.php [hunk 2]
```diff
@@ -76,7 +76,9 @@ abstract class Base extends TestCase
 
     protected const MULTIPART_RESPONSES = [
         'abc',
-        'd80e7e6999a3eb2ae0d631a96fe135a4' #
+        'd80e7e6999a3eb2ae0d631a96fe135a4',
+        '{"key": "value", "key2": "value2"}',
+        'value',
```
Comment (Meldiron) [inline tests/Base.php:81]: A bit scared to use such simple term. TO be extra safe, lets do `stringPayloadValue` or similar

File: tests/resources/spec.json [hunk 3]
```diff
@@ -1666,6 +1663,58 @@
...
+		  "description": "",
+      "responses": {
+        "200": {
+          "description": "Multipart",
+          "schema": {
+            "$ref": "#\/definitions\/multipart"
+          }
+        }
+      },
+		  "x-appwrite": {
```
Comment (Meldiron) [inline tests/resources/spec.json:1685]: Formatting seems wrong here

File: tests/languages/web/index.html [hunk 4]
```diff
@@ -1,202 +1,204 @@
...
-
-      client.setProject("console");
-      client.setEndpointRealtime("ws://demo.appwrite.io/v1");
-
-      client.subscribe("tests", (event) => {
-        responseRealtime = event.payload.response;
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Meldiron) [inline tests/languages/web/index.html:6]: Lets revert formatting changes please in index.html

File: tests/Base.php (hunk 2 above)
Comment (loks0n) [reply tests/Base.php]: Changed

File: tests/languages/web/index.html (hunk 4 above)
Comment (loks0n) [reply tests/languages/web/index.html]: Done
